src/main/resources/api-swagger-template.yaml

```diff
@@ -0,0 +1,67 @@
+securityDefinitions:
```

Comment:
File needs a comment explaining its purpose.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/59/commits/b327a799b261096bd67fcaf0d680d92b1090fc2a